src/styles/colors.js

```diff
@@ -9,3 +9,18 @@ export const PRIMARY_WEAK = '#73dbff';
 export const PRIMARY_STRONG = '#17bcde';
 
 export const GREEN = '#b5ce4d';
+
+export const ARTICLES = {
```

Comment:
IMHO: if we keep that in `styles/colors.js`, i think it makes sense to make their usage similar to the other colors. Especially, since we don't know how the colors are used in other exercises.

But we can also keep it this way for now, can always refactor later when we need them somewhere else.